### DISCUSSION TITLE
Is there anyway to have a "User" Type Col saved in Clear Text?

### DISCUSSION THREAD

**Discussion by mcancell** [Budibase/budibase#12915] [Help]
I am trying to take advantage of the User Type column. However, it stores a hash of the user (email) vs. the clear text email. Is there anyway around this?

**Comment by melohagan** [comment]
Hey @mcancell 

As far I understand we don't hash the user email. The ID that is saved is a randomly generated guid. 

You could save/access the email address with the following binding: `{{ Repeater.Employees.UserF.0.email }}` where 'UserF' is a user field of the 'Employees' table

**Reply by mcancell** [reply]
Excellent. Thank you. I'll try this later today.
